src/wallet/wallet.h

```diff
@@ -837,7 +850,8 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
      * completion the coin set and corresponding actual target value is
      * assembled
      */
-    bool SelectCoinsMinConf(const CAmount& nTargetValue, int nConfMine, int nConfTheirs, uint64_t nMaxAncestors, std::vector<COutput> vCoins, std::set<CInputCoin>& setCoinsRet, CAmount& nValueRet) const;
+     // TODO: Change the hard coded change_size when we aren't only using P2PKH change outputs
```

Comment:
if we're going to change it later to something without a default/dynamic value, maybe just get rid of the default arg and pass it each time.

Reply:
this is getting more important now with segwit, and the fact that we have two related optional arguments